index.html

```diff
@@ -229,6 +229,6 @@ <h4 class="popup__type">Квартира</h4>
       <button type="button" class="error__button">Попробовать снова</button>
     </div>
   </template>
-
+  <script src="/Users/aryey/Desktop/Development/Projects/1864601-keksobooking-24/js/main.js"></script>
```

Comment:
```suggestion
  <script src="./js/main.js"></script>
```
нужно указывать относительный путь от файла